code-cademy.html

```diff
@@ -1,9 +1,239 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
 <head>
+	<meta charset="utf-8">
 	<title>codecademy</title>
+	<meta name="viewport" content="width=device-width">
+	<link rel="stylesheet" type="text/css" href="code-cademy.css">
+	<link href="https://use.fontawesome.com/releases/v5.0.7/css/all.css" rel="stylesheet">
 </head>
 <body>
 
+	<header>
+		<nav class="site-nav">
+			<ul>
+				<li><a href="">Catalog</a></li>
+				<li><a href="">Learn From Home</a></li>
+				<li><a href="">Pricing</a></li>
+				<li><a href="">For Business</a></li>
+
+			<ul class="nav">
+				<li><a href=""><b>Log in</b></a></li>
+				<li><a href=""><strong>Sign up</strong></a></li>
+			</ul>
+
+			</ul>
+		</nav>
+			<div class="fix"></div>
+	</header>
```

Comment:
You have good indentation here as it is obvious that `nav` is a child element of `header` and `ul` child element of `nav` and so on.